File: codesignal/arcade/Intro/JS/addBorder.js
function addBorder(picture) {
    let startEndStars = ""
    for(let i = 0; i < picture.length; i++) picture[i] = "*" + picture[i] + "*"
    for(let i = 0; i < picture[0].length; i++) startEndStars+= "*"
    picture.unshift(startEndStars)
    picture.push(startEndStars)
    return picture
}

File: codesignal/arcade/Intro/Csharp/addBorder.cs
string[] addBorder(string[] picture) {
    string stars = "";
    List<string> result = new List<string>();
    
    for(int i = 0; i < picture[0].Length + 2; i++) stars += '*';
    result.Add(stars);
    
    for(int i = 0; i < picture.Length; i++){
        result.Add("*" + picture[i] + "*");
    }
    
    result.Add(stars);
    
    return result.ToArray();
}
